app/screens/AppNavigator/screens/Settings/SettingsScreen.tsx

```diff
@@ -69,6 +69,7 @@ function RowNetworkItem (props: { network: EnvironmentNetwork }): JSX.Element {
 
   return (
     <TouchableOpacity
+      testID={`button_network_${props.network.replace(' ', '')}`}
```

Comment:
```suggestion
      testID={`button_network_${props.network}`}
```

Need to update this with #342